pkg/event/dispatcher.go

```diff
@@ -111,11 +95,11 @@ func (ed *QueueEventDispatcher) flushEvents() {
 
 	retryCount := 0
 
-	ed.metrics.SetQueueSize(ed.eventQueue.Size())
+	ed.stats.Set("queueSize", int64(ed.eventQueue.Size()))
```

Comment:
Do these keys need to be namespaced by package? Otherwise we'd run into collisions with key names

Reply:
yes, they can be. Sidedoor has the prefix variable to set the prefix. I can have the same implementation here